src/fmu/ensemble/realization.py

```diff
@@ -384,6 +384,15 @@ def load_csv(self, localpath, convert_numeric=True, force_reread=False):
                 else:
                     dtype = str
                 dframe = pd.read_csv(fullpath, dtype=dtype)
+                if "REAL" in dframe:
+                    dframe.rename(columns={"REAL": "REAL_ORIG"}, inplace=True)
+                    logger.warning(
+                        (
+                            "Loaded file %s already had the column REAL, "
+                            "this was renamed to REAL_ORIG"
```

Comment:
Could these potentially become different, at some point? Why keep the REAL_ORIG, why not just drop it?

Reply:
They would be different if the user has a CSV file where REAL has nothing to do with a realization at all.

Moving instead of deleting will be a nuisance to some, and critical to others - not easy what is best. You prefer deletion?

Reply:
Not sure, it just seems a bit redundant. Not a big deal.